aiohttp/connector.py

```diff
@@ -467,30 +481,57 @@ def family(self):
         return self._family
 
     @property
+    def cache_dns(self):
```

Comment:
use_dns_cache is the best, sure.
Thanks for suggestion

On Sat, Jun 20, 2015 at 4:37 PM, Alexander Shorin <notifications@github.com>
wrote:

> In aiohttp/connector.py
> <https://github.com/KeepSafe/aiohttp/pull/415#discussion_r32884576>:
>
> > @@ -467,30 +481,57 @@ def family(self):
> >          return self._family
> >
> >      @property
> > +    def cache_dns(self):
>
> Worth to. Btw, on second thought I found myself incorrect about the name:
> seems like this property is used as True/False flag in order to define will
> dns cache be used or not, right? Then use_dns_cache should be
> semantically better and won't cause confusion about if it some sort of
> cache mapping or not. What do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KeepSafe/aiohttp/pull/415/files#r32884576>.
>



-- 
Thanks,
Andrew Svetlov